site/generate.sh

```diff
@@ -40,7 +40,7 @@ echo '<?php $rules = array( ' > "$RULE"
 #
 # Looking at statistics like http://stats.jenkins-ci.org/jenkins-stats/svg/201409-jenkins.svg,
 # I think three or four should be sufficient
-for v in 1.554 1.565 1.580 1.596; do
+for v in 1.554 1.565 1.580 1.596 1.609; do
```

Comment:
Should we also remove the oldest one at this point?
The comment above seems to suggest we should keep three or four in this list.